cmd/repo.go

```diff
@@ -279,18 +282,23 @@ func getArtifactsNotIndexed(index repo.Index, distributedFiles string) repo.Inde
 	filesNotIndexed := getFilesNotIndexed(index, distributedFiles)
 
 	artifactsNotIndexed := make(repo.Index, 0)
+
 	for _, file := range filesNotIndexed {
 		artifactInfo := new(repo.ArtifactInfo)
 		artifactInfo.Filename = filepath.Base(file.Name())
 
 		if err := artifactInfo.SetInfoFromFileName(artifactInfo.Filename); err != nil {
-			continue
+			artifactInfo.Name = artifactInfo.Filename
+			artifactInfo.Version = "unknown"
```

Comment:
Are you sure that "unknown" don't broke any version checking???

Reply:
What do you think could be a good version for this artifacts?